itests/util/src/main/java/org/apache/hadoop/hive/ql/QOutProcessor.java

```diff
@@ -358,6 +349,7 @@ private ArrayList<Pair<Pattern, String>> initPatternWithMaskComments() {
         add(toPatternPair(PATH_HDFS_WITH_DATE_USER_GROUP_REGEX, String.format("%s %s$3$4 %s $6%s",
             HDFS_USER_MASK, HDFS_GROUP_MASK, HDFS_DATE_MASK, HDFS_MASK)));
         add(toPatternPair(PATH_HDFS_REGEX, String.format("$1%s", HDFS_MASK)));
+        add(toPatternPair("(.*totalSize\\s*)\\d+(.*)", "$1#Masked#$2"));
```

Comment:
I'm afraid that "$1#Masked#$2" is supposed to introduce additional flakiness when we mask stuff like "999" vs "1001", instead "$1#Masked#" is the proper thing to do, I remember @SourabhBadhya investigated this while backporting HIVE-27819 to Cloudera's downstream
correct way for example:https://github.com/apache/hive/pull/4824/files#diff-23f10da4cb3c3f2bad832475c7b3f9c21e805ff8d3215856d6468bf449b7009dR3

Reply:
Yes, this can also problems when the total size is near a n-digit number and an (n+1)-digit number. Using the right rule can avoid such flaky failures.

Reply:
Thanks for the info! Fixed.